server/views/list.pug

```diff
@@ -1,6 +1,22 @@
 extends layout
 
 block one-column
+  - var i = 1;
+  nav(aria_label="Page navigation")
+    ul.pagination
+      li
+        a(href=`/storage/list/${index-1}`, aria-label="Previous")
+          span(aria-hidden="true") &laquo;
+        while i <= nPage
+          li
+            if index == i
+              a(href=`/storage/list/${i}`, id='current')=i++
```

Comment:
Tu peux utiliser http://getbootstrap.com/components/#disabled-and-active-states plutôt que du Javascript s'il-te-plait :) ?